src/theme/components/card.ts

```diff
@@ -128,6 +128,8 @@ const CardActions = {
 const CardContent = {
   baseStyle: {
     marginBottom: "xs",
+    width: "100%",
```

Comment:
This was there to enable the three-column card layout, but I think the cards need to be redesigned